homework1/homework1.css

```diff
@@ -0,0 +1,17 @@
+* {margin: 0; padding: 0;}
+body {
+    background-color: blue;
+}
+
+.one {
+    text-align: center;
+    align-items: center;
+}
+h3 {
```

Comment:
```css
.one h3 {
  ....
}
```

Selectors should be precise (Css selector là phải chuẩn, nó nên dc select theo thẻ cha, để tránh ảnh hưởng các thẻ h3 ở ngoài div#one)

Reply:
thank you. i got it